lib/mathlib.cpp

```diff
@@ -356,8 +356,6 @@ MathLib::bigint MathLib::characterLiteralToLongNumber(const std::string& str)
     // is implementation-defined.
     // clang and gcc seem to use the following encoding: 'AB' as (('A' << 8) | 'B')
     const std::string& normStr = normalizeCharacterLiteral(str);
-    if (normStr.empty())
-        throw InternalError(0, "Internal Error. MathLib::characterLiteralToLongNumber: Unhandled char constant '" + str + "'.");
```

Comment:
Maybe keep that security check? normalizeCharacterLiteral() is non-trivial, maybe it can still fail somehow or after some future modifications.